main.go

```diff
@@ -159,7 +159,10 @@ func (b *bot) onMessageCreate(s *discordgo.Session, m *discordgo.MessageCreate)
 		}
 		b.karmaGet(m.Message, target)
 		return
-	} else if idx, ok := mee6inform[parts[0]]; ok {
+	} else if parts[0] == "!mod" {
+        b.requestMod(s, m.Message, parts[1:])
```

Comment:
This is a _nit_ (i.e. non-essential suggestion).

There's no need to pass the session `s`, it'll be the same as the session stored in [`b.discord`](https://github.com/Woovie/iggystats/blob/2aaf395681f8fab1875234c85b0dd1f2a8f5b7ad/main.go#L48).

```suggestion
        b.requestMod(m.Message, parts[1:])
```